imports/plugins/core/ui/client/components/table/sortableTable.js

```diff
@@ -253,6 +252,21 @@ class SortableTable extends Component {
     return className;
   }
 
+  /**
+   * @name displayNoResultsFound()
+   * @method
+   * @summary This function displays a 'No Results Found' when there is no data to populate the table
+   * @return {node} returns a JSX node or empty string
+   */
+  displayNoResultsFound() {
+    let displayText = "";
+    if (this.getTableData() === 0) {
+      displayText = <span className="sortableTable-noDataText">{this.props.noDataMessage}</span>;
+      return displayText;
```

Comment:
You don't need the return inside the `if` block.

Reply:
Changes requested fixed @zenweasel